recipes/gargammel/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.1.2" %}
+
+package:
+  name: gargammel
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/grenaud/gargammel/archive/{{ version }}.tar.gz
+  sha256: c4b5ca081087fa5ecf086dbd49a4db0fb3c7d2d3d13eb30bdfc96b964661c3c1
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+    - cmake
+    - gsl
+    - gnu-wget
+    - openssl 1.0.2n
+  run:
+    - zlib
+    - openssl 1.0.2n
+    - perl
+    - gsl
+    - art
+    - python=3.6
+
+about:
+  home: https://github.com/grenaud/gargammel
+  license: GPL
+  license_file: LICENSE
+  summary: Tool for simulating ancient DNA datasets
+
+test:
+  commands:
+    - gargammel --help    
```

Comment:
Could you also test running the binaries that the Perl script calls? The original script tested whether it could find those binaries, but the patched version doesn’t anymore, so while I trust that it works at the moment since you did some testing locally, it would be easy to miss problems in a future version of the recipe.

Reply:
Yes, will do. `art` definitely worked because I saw log output, but will check the others.

That said, wierdly, now after cleaning up my conda environments, I've found there is a unlisted depency of `samtools`.

But when I try locally re-building the package with the updated `meta.yaml`, and try `conda install . --use-local`, it doesn't pick `samtools` up...

Reply:
`art` is the one you don’t actually need to test because that is now pulled in as an external dependency.

Good you caught the samtools dependency!